lib/engine/clang/src/this_engine.cpp

```diff
@@ -31,8 +31,10 @@ namespace metashell
       {
         try
         {
-          return regex_search(process::run(clang_, {"-v"}, "").standard_error,
-                              std::regex("^clang version "));
+          auto standard_error = process::run(clang_, {"-v"}, "").standard_error;
+          return regex_search(standard_error, std::regex("^clang version ")) ||
```

Comment:
Why two regexes instead of a single one covering both cases?

Reply:
No particular reason. That seemed the easiest without checking the reference. I merged them together now.

Reply:
Thanks